lib/perl/Genome/SoftwareResult/ImportedFile.pm

```diff
@@ -0,0 +1,26 @@
+package Genome::SoftwareResult::ImportedFile;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::SoftwareResult::ImportedFile {
+    is => 'Genome::SoftwareResult::StageableSimple::SingleFile',
+    has_input => [
+        source_file_path => {
+            is => 'Path',
```

Comment:
When we "import" files into the system it's better to use an MD5 of the file to uniquely identify it rather than the path from which it came.

`Genome::Model::Tools::DetectVariants2::Result::Manual` is one example of doing this.  The InstrumentData import process is another.